src/planning_core/src/planning_process/planning_process.cpp

```diff
@@ -10,12 +10,148 @@ namespace Planning {
     // Read config file for planning process
     configReaderProcess = std::make_unique<ConfigReader>();
     configReaderProcess->readProcessConfig();
-    auto obsDis = configReaderProcess->getProcess().obs_dis_;
+    obsDis = configReaderProcess->getProcess().obs_dis_;
+
+    // create PNCMap & Global Path Client
+    pncMapClient = this->create_client<base_msgs::srv::PNCMapService>("pnc_map_server");
+    globalPathClient = this->create_client<base_msgs::srv::GlobalPathService>("global_path_server");
   }
 
   boolean PlanningProcess::process()
   {
+    // set 1 second timer to wait rviz2 & xacro to be ready
+    rclcpp::Rate rate(1s);
```

Comment:
This is wrong. For rate constructor the input should be 1.0 but not 1s.
```suggestion
    rclcpp::Rate rate(1.0);
```